src/v2/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -233,7 +229,7 @@ export class ShippingRoute extends Component<ShippingProps, ShippingState> {
             }
           }, // onError
           this.props.me, // me
-          () => this.setState({ showModal: false }) // closeModal
```

Comment:
I investigated if this argument could be completely removed from the function being called, but it is still needed for other code paths.

Reply:
Yes, you are right. Now modals state based in SavedAddresses.tsx